swift/ITSClient/Sources/ITSMobility/Observers/RoadAlarm/RoadAlarm.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+
+/// The representation of a road alarm.
+public struct RoadAlarm: Sendable {
+    /// The unique identifier.
+    public let id: String
+    /// The cause.
+    public let cause: CauseType?
+    /// The longitude.
+    public let longitude: Double
+    /// The latitude.
+    public let latitude: Double
+    /// The road alarm timestamp.
+    public let timestamp: Date
+    /// The road alarm lifetime.
+    public let lifetime: TimeInterval
+    /// The underlying DENM object.
+    public let underlyingDENM: DENM
+
+    var expirationDate: Date {
+        timestamp.addingTimeInterval(lifetime)
+    }
+
+    var isExpired: Bool {
+        Date() > expirationDate
+    }
+
+    /// Initializes a `RoadAlarm`.
+    /// - Parameters:
+    ///   - id: The unique identifier.
+    ///   - cause: The cause.
```

Comment:
The sub-cause is also used quite often. It'sd a PITA to handle and define (see commit 5ad9a11071d157043cb197b30bde9cd75bc4df88), but it is quite important to have in the SDK.

Reply:
It's not really related to this PR, I will create an issue and add it in another PR if you agree.